aws_iam_role_packer_instance_profile.tf

```diff
@@ -0,0 +1,131 @@
+resource "aws_iam_role" "codebuild_instance_role" {
+  name = "CODEBUILD-PACKER-IAM-ROLE"
```

Comment:
@andysingleton can we make the name a bit more modular? Because I think we will have an issue in AWS if we re-run this Codebuild module multiple times as IAM roles/polices/names need to be unique.